sdk/keyvault/keyvault-certificates/test/internal/apiVersionParameter.spec.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import { createSandbox } from "sinon";
+import { CertificateClient } from "../../src";
+import { LATEST_API_VERSION } from "../../src/certificatesModels";
+import { HttpClient, WebResourceLike, HttpOperationResponse, HttpHeaders } from "@azure/core-http";
+import { ClientSecretCredential } from "@azure/identity";
+import { env } from "@azure/test-utils-recorder";
+
+describe("The Certificates client should set the apiVersion", () => {
+  const keyVaultUrl = `https://keyVaultName.vault.azure.net`;
+  const mockHttpClient: HttpClient = {
+    async sendRequest(httpRequest: WebResourceLike): Promise<HttpOperationResponse> {
+      return {
+        status: 200,
+        headers: new HttpHeaders(),
+        request: httpRequest,
+        parsedBody: {
+          id: `${keyVaultUrl}/certificates/certificateName/id`,
+          attributes: {}
+        }
+      };
+    }
+  };
+
+  it("it should default to the latest API version", async function() {
+    const sandbox = createSandbox();
```

Comment:
should we do a beforeEach/afterEach to create and restore the sandbox?

Reply:
Can do!

Reply:
Let me know how the latest updates look 🌞